src/components/SVGViewer/stories/click.md

```diff
@@ -0,0 +1,29 @@
+# Item click
+
+<!-- STORY -->
+
+#### Description:
+
+If to click on any equipment name will be returned
+
+#### Usage:
+
+```typescript jsx
+import 'antd/dist/antd.css';
+
+import React from 'react';
+import { SVGViewer } from '@cognite/gearbox';
+
+function ExampleComponent(props) {
+
+  return (
+    <div style={{ height: '100vh' }}>
+      <SVGViewer
+        documentId={5185355395511590}
+        handleItemClick={handleItemClick}
+      />
```

Comment:
Better to provide function description inside example, something like:
```typescript
function ExampleComponent(props) {

  const handleItemClick = (metadataNode: HTMLElement): void => { ... }

  return ( ... )
}
```
So user hasn't to switch to "Full description" page to check params of **handleItemClick** callback